extensions/notebook/src/extension.ts

```diff
@@ -133,7 +133,7 @@ async function openNotebook(): Promise<void> {
 			vscode.window.showTextDocument(doc);
 		}
 	} catch (err) {
-		vscode.window.showErrorMessage(err);
+		vscode.window.showErrorMessage(err.message);
```

Comment:
Should we be using the getErrorMessage utility? Are we always guaranteed to get Error here, and not a string?